packages/api/helper/mailer.js

```diff
@@ -171,12 +171,31 @@ function sendHIPAAProvider(nomination) {
   .catch((err) => console.log(err))
 }
 
+function sendSurveySocialWorker(nomination) {
+  email
+    .send({
+      template: 'surveySocialWorker',
+      message: {
+        from: 'formmaster@keepswimmingfoundation.org',
```

Comment:
Would you mind creating a constant for `formmaster@keepswimmingfoundation.org` and updating all the code here to use that instead of copying this string everywhere?

Reply:
Sure!